src/Frontend/Page/Overview.hs

```diff
@@ -119,7 +118,7 @@ instance ToHtml PageIdeasOverview where
                     a_ [href_ U.Broken] "Umgebung"
         div_ [class_ "m-shadow"] $ do
             div_ [class_ "ideas-list"] . for_ ideaAndNumVoters $ \(idea, numVoters) ->
-                ListItemIdea True Nothing numVoters idea ^. html
+                ListItemIdea IdeaInIdeasOverview Nothing numVoters idea (user ^. userRole) ^. html
```

Comment:
My idea was:

```
newtype DisplayCtx = DisplayCtx User
```

...  and then we pass that to all pages that change appearance based on who visits them.  (I would have put user id and role there, but the important point is to have a name dedicated to this.)